internal/cmd/cmds.go

```diff
@@ -151,6 +152,12 @@ func (cmd *DiceDBCmd) Key() string {
 	return c
 }
 
+func GetNilRes() *CmdRes {
+	return &CmdRes{R: &wire.Response{
+		Value: &wire.Response_VNil{VNil: true},
+	}}
+}
+
```

Comment:
Why this function? why can't we reuse `cmdResNil`?

Reply:
`cmdResNil` was a shared object used across `cmd` package. At time multiple go-routines were updating the same object as part of `r.R.Attrs.Fields["fingerprint"] = structpb.NewStringValue(strconv.FormatUint(uint64(c.Fingerprint()), 10))` or `proto.Marshal` which was causing concurrency error.